src/Components/TodoList.js

```diff
@@ -28,12 +28,12 @@ function TodoList({ todoList, onRemoveTodo }) {
   );
 }
 TodoList.propTypes = {
-    todoList: PropTypes.arrayOf(
-        PropTypes.shape({
-            id: PropTypes.number.isRequired,
-            title: PropTypes.string.isRequired,
-        })
-    ).isRequired,// PropTypes for the 'todoList' prop
+  todoList: PropTypes.arrayOf(
+    PropTypes.shape({
+      id: PropTypes.string,
+      title: PropTypes.string,
```

Comment:
nice use of shape! so the `id` and the `title` aren't `required`?

Reply:
Since you're choosing to specify the entire object `shape`, you should also include a required value for `createdDate`!

I had no idea how this should be done so I googled it, and you would define it as: 

```suggestion
      id: PropTypes.string,
      title: PropTypes.string,
      createdTime: PropTypes.instanceOf(Date).isRequired,
```

Reply:
> nice use of shape! so the `id` and the `title` aren't `required`?

I kept getting the propType error message when I made those values `required`, that was why I changed that. It was the only fix I could come up with

Reply:
dmed you!